Maskbook/Extension/Entity/PersonaRecord+Identifier.swift

```diff
@@ -17,4 +17,23 @@ extension PersonaRecord {
         }
         return identifier
     }
+    
+    var privateKeyBase64String: String {
+        let originPersona = Persona(fromRecord: self)
+        return originPersona?.privateKey?.privateKeyBase64String ?? ""
+    }
+    
+    var qrCodeString: String {
+        var qrCodeString = "mask://persona/privatekey/" + privateKeyBase64String
```

Comment:
Should we use the following instead? This way, we can avoid unnecessary triggering the getter for `privateKeyBase64String `, and string interpolation  here may be more efficient than `+` function. the implementation will be more efficient.
```
        var qrCodestr = {
            if let mnemonic = self.mnemonic?
                .data(using: .utf8)?
                .base64EncodedString() {
                return "mask://persona/mnemonic/\(mnemonic)"
            } else {
                return "mask://persona/privatekey/\(privateKeyBase64String)"
            }
        }()
```